tests/foreman/api/test_docker.py

```diff
@@ -1370,7 +1372,14 @@ def test_positive_delete(self):
         @Assert: The docker containers are deleted in local and external
         compute resources
         """
-        for compute_resource in (self.cr_internal, self.cr_external):
+        cr_interfaces = [self.cr_external]
+        # there is some bugs affecting docker internal interface on RHEL7
+        if get_host_os_version().startswith('RHEL7'):
+            bugs_opened = [
+                bz for bz in [1366573, 1414797] if bz_bug_is_open(bz)]
+            if not bugs_opened:
```

Comment:
I would not use a list comprehension for only 2 items..

```python
if not bz_bug_is_open(1366573) and not bz_bug_is_open(1414797):
    ....
```
is simpler and better to understand than the list comprehension version, I would use the comprehension only if we had a large list of Bzs.

What do you think?